### PR TITLE
Fix some more samples and add them to build

### PR DESCRIPTION
**Pull request by bdilly**
There is a warning fixed by ibriano on pr #720

Signed-off-by: Bruno Dilly <bruno.dilly@intel.com>

### CODE REVIEW

File: src/samples/flow/unix-socket/byte-writer.fbp [hunk 1]
```diff
@@ -32,6 +32,6 @@
 # On a timeout (every second) create a byte and send to the socket.
 # See byte-reader.fbp
 
-Timer(timer:interval=1000) OUT -> INC Acc(int/accumulator:setup_value=val:0|min:0|max:254:step:1)
+Timer(timer:interval=1000) OUT -> INC Acc(int/accumulator:setup_value=val:0|min:0|max:254|step:1)
```
Comment (lpereira) [inline src/samples/flow/unix-socket/byte-writer.fbp]: Was this a syntax error as far as the parser was concerned?

Comment (bdilly) [inline src/samples/flow/unix-socket/byte-writer.fbp]: Yeah, a syntax error.
This sample wasn't running or building.

**Comment by bdilly** [comment]
It's building fine on my machine, on semaphore and tipbot.
We're just adding as dependencies of these samples node types that are built as modules by default. Maybe you're buliding console as module or something like that?
I'll see this keyboard dependency.
Thanks

**Comment by bdilly** [comment]
@ceolin 

Ok, it's passing on all these machines because they don't have enough dependencies to build http-server, I've just checked the log.

I'll send a v2 fixing keyboard dependency but if something else goes wrong, please paste a log so I can fix all issues.

Thanks

**Comment by bdilly** [comment]
pr #733

**Pull request closed without merging**